### PR TITLE
[Master] - Fix links post learn page revamp

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Fix links post learn page revamp.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by sm1990 — DISMISSED** [review]

File: examples/c2c-docker-deployment/c2c_docker_deployment.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Docker
 
 Ballerina supports generating Docker artifacts from code without any additional configuration. This simplifies the experience of developing and deploying Ballerina code in the cloud. Code to Cloud builds the containers and required artifacts by deriving the required values from the code. If you want to override the default values taken by the compiler, you can use a `Cloud.toml` file.
-For more information, see [Code to Cloud Deployment](/learn/run-in-the-cloud/code-to-cloud/code-to-cloud-deployment/).
+For more information, see [Code to Cloud Deployment](/learn/code-to-cloud/code-to-cloud-deployment/).
```
Comment (sm1990) [inline examples/c2c-docker-deployment/c2c_docker_deployment.md:4]: Remove `code-to-cloud/`

File: examples/c2c-k8s-deployment/c2c_k8s_deployment.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Kubernetes
 
 Ballerina supports generating Kubernetes artifacts from code without any additional configuration. This simplifies the experience of developing and deploying Ballerina code in the cloud. Code to Cloud builds the containers and required artifacts by deriving the required values from the code. If you want to override the default values taken by the compiler, you can use a `Cloud.toml` file.
-For more information, see [Code to Cloud Deployment](/learn/run-in-the-cloud/code-to-cloud/code-to-cloud-deployment/).
+For more information, see [Code to Cloud Deployment](/learn/code-to-cloud/code-to-cloud-deployment/).
```
Comment (sm1990) [inline examples/c2c-k8s-deployment/c2c_k8s_deployment.md:4]: Remove `code-to-cloud/`

File: examples/docker-hello-world/docker-hello-world.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Docker - Hello world
 
 Ballerina supports generating Docker artifacts from code without any additional configuration. This simplifies the experience of developing and deploying Ballerina code in the cloud. Code to Cloud builds the containers and required artifacts by deriving the required values from the code. If you want to override the default values taken by the compiler, you can use a `Cloud.toml` file.
-For more information, see [Code to Cloud Deployment](/learn/run-in-the-cloud/code-to-cloud-deployment/).
+For more information, see [Code to Cloud Deployment](/learn/code-to-cloud-deployment/).
```
Comment (sm1990) [inline examples/docker-hello-world/docker-hello-world.md:4]: Remove `code-to-cloud/`

File: examples/c2c-docker-deployment/c2c_docker_deployment.md (hunk 1 above)
Comment (praneesha) [reply examples/c2c-docker-deployment/c2c_docker_deployment.md]: Fixed.

File: examples/c2c-k8s-deployment/c2c_k8s_deployment.md (hunk 2 above)
Comment (praneesha) [reply examples/c2c-k8s-deployment/c2c_k8s_deployment.md]: Fixed.

**Review by sm1990 — APPROVED** [review]

**Pull request merged**